### ISSUE TITLE
[BUGS] Minor frontend bugs on the new StreamETH Page

### ISSUE THREAD

**Issue by xannyxs** [streamethorg/streameth-platform#361]
**Describe the bug**
Download button is not aligned with download button on phone & laptop.

**To Reproduce**
1. Open [link](https://streameth.org/watch?event=base_event&session=onchain_music_featuring_the_jackson_five) on your phone or laptop
2. See error

**Expected behavior**
Should be aligned

**Screenshots**
<img width="296" alt="Screenshot 2024-01-26 at 14 54 43" src="https://github.com/streamethorg/streameth-platform/assets/56308735/c7b3dac4-4c93-45b9-8776-cac4eb69f54e">

---

**Describe the bug**
Index of page count is incorrect when there are no videos found via the searchbar

**To Reproduce**
1. Open [link](https://streameth.org/archive?searchQuery=coolddjwqdmqwmdqwdqw) on your phone or laptop
2. See error in right corner

**Expected behavior**
Page index should be hidden or the 0 should be 1

**Screenshots**
<img width="296" alt="Screenshot 2024-01-26 at 14 59 04" src="https://github.com/streamethorg/streameth-platform/assets/56308735/5de90b08-c2b6-4781-833e-b607081bad6b">

---

**Describe the bug**
State of hamburger is incorrect on phone

**To Reproduce**
1. Go to the homepage on your phone.
2. See error on the top of the page

**Expected behavior**
The hamburger should be closed when loading the page, instead of open.

**Screenshots**
<img width="981" alt="Screenshot 2024-01-26 at 15 00 40" src="https://github.com/streamethorg/streameth-platform/assets/56308735/38562c94-d596-4e42-b53f-7d23b03192a0">

---

**Describe the bug**
The related videos are section is too big and has its own scrollbar, even though it is not scrollable.

**To Reproduce**
1. Go to this [link](https://streameth.org/watch?event=ethberlin_2022&session=humanity_30_beyond_blockchains_planetary_systems__infinite_games)
2. See error on the `related`

**Expected behavior**
Should be the size of YouTube suggestions (do no know the exact size). Should not be scrollable via related but page wide.

**Screenshots**
<img width="528" alt="Screenshot 2024-01-26 at 15 02 49" src="https://github.com/streamethorg/streameth-platform/assets/56308735/78f52d1a-28cf-4ec7-852b-99235855c9fa">

---

**Describe the bug**
Alignment of `related` and `navbar` buttons is correct, but it should be aligned with the background. (Or show no background at all)

**To Reproduce**
1. Go to this [link](https://streameth.org/watch?event=ethberlin_2022&session=humanity_30_beyond_blockchains_planetary_systems__infinite_games)
2. See error on the right corner of the page.

**Expected behavior**
The related section should align with the buttons of the `Navbar`.

**Screenshots**
<img width="441" alt="Screenshot 2024-01-26 at 15 08 32" src="https://github.com/streamethorg/streameth-platform/assets/56308735/14729a04-099f-4e5c-b853-702ee337c1f5">

**Comment by xannyxs** [comment]
@pabloVoorvaart Would you agree with the bugs?

**Comment by xannyxs** [comment]
@ashuydv, here you can find some frontend bugs. Please wait for @pabloVoorvaart for approvement of the issue before starting

**Comment by greatsamist** [comment]
> **Describe the bug** Download button is not aligned with download button on phone & laptop.
> 
> **To Reproduce** Steps to reproduce the behavior:
> 
> 1. Open [link](https://streameth.org/watch?event=base_event&session=onchain_music_featuring_the_jackson_five) on your phone or laptop
> 2. See error

Bug already fixed and merged into develop

**Comment by greatsamist** [comment]
Describe the bug
#### State of hamburger is incorrect on phone

Also fixed and merged into develop

**Comment by ashuydv** [comment]
ok @xannyxs

**Comment by xannyxs** [comment]
@greatsamist, what do you think if the last two bug fixes? Is that okay by you?

**Comment by greatsamist** [comment]
> **Describe the bug** Index of page count is incorrect when there are no videos found via the searchbar
 
> **Expected behavior** Page index should be hidden or the 0 should be 1
### The page count should be 1 of 1, if no video is found.


> **Describe the bug** The related videos are section is too big and has its own scrollbar, even though it is not scrollable.
### The player takes 70% of the page width,  leaving 30% for related videos which is fine by me.  we just need to remove the scrollbar on related videos

> **Describe the bug** Alignment of `related` and `navbar` buttons is correct, but it should be aligned with the background. (Or show no background at all)

### This needs to be fix on large screens, alignment is fine on smaller screens

**Comment by ashuydv** [comment]
Hey, i set up the .env file, but now i'm getting error of NEXT_PUBLIC_API_URL, let me know how can i fix this ? 

![image](https://github.com/streamethorg/streameth-platform/assets/67309431/c3356fc2-90c0-4407-9abb-7e08fc327108)

**Comment by pabloVoorvaart** [comment]
hey there @ashuydv can you share your telegram maybe, i can help you get setup via dm, thanks

**Comment by ashuydv** [comment]
https://t.me/ashuydv16, hey you can ping me here ! sorry for the delay, had some health issues !
